models/membership_test.go

```diff
@@ -529,6 +529,7 @@ var _ = Describe("Membership Model", func() {
 				Expect(dbMembership.PlayerID).To(Equal(player.ID))
 				Expect(dbMembership.ClanID).To(Equal(clan.ID))
 				Expect(dbMembership.Approved).To(BeTrue())
+				Expect(dbMembership.ApprovedAt).To(BeNumerically("~", util.NowMilli(), 1000))
```

Comment:
This is a new assertion added to ensure `ApprovedAt` is being updated correctly.